glean_parser/lint.py

```diff
@@ -262,6 +262,36 @@ def check_expired_metric(
         yield ("Metric has expired. Please consider removing it.")
 
 
+def check_redundant_ping(
+    pings: Iterable[pings.Ping]
+) -> LintGenerator:
+    '''
+    Check if the pings contains 'ping' as the prefix or suffix, or 'ping' or 'custom'
+    '''
+    ping_words = sorted([_split_words(ping.name) for ping in pings])
+
+    if len(ping_words) != 0:
+        ping_first_word = ping_words[0]
+        ping_last_word = ping_words[-1]
+
+        if ping_first_word == "ping":
+            yield (
+                "The prefix 'ping' is redundant."
+            )
```

Comment:
Best to run `make lint`, which will also reformat this (and following lines) to

```
yield ("The prefix 'ping' is redundant.")
```

(to reduce vertical space)

Reply:
Done